Gopkg.toml

```diff
@@ -44,7 +45,8 @@ required = [
   version = "=1.11.1"
 [[constraint]]
   name = "github.com/go-swagger/go-swagger"
-  version = "=0.20.1"
+  source = "github.com/Percona-Lab/go-swagger"
+  version = "=0.20.1-percona2"
```

Comment:
Let's add a short comment and a link to PR explaining when we can switch to the upstream.